release.js

```diff
@@ -121,10 +121,15 @@ function publish() {
   console.log(gitPushTags);
   execSync(gitPushTags);
 
+  // remove fixture's node_nodules to avoid stuck on the next step
+  const cleanFixturesNodeModules = `git clean -dfx packages/test-support/fixtures`;
```

Comment:
This works for now. We can also just exclude this with `files` in package.json

Reply:
Updated to solve it this way :)